src/bun.js/webcore/S3File.zig

```
@@ -147,6 +147,19 @@ pub fn write(globalThis: *jsc.JSGlobalObject, callframe: *jsc.CallFrame) bun.JSE
     switch (path_or_blob) {
         .path => |path| {
             const options = args.nextEat();
+            var mode: ?bun.Mode = null;
+
+            // Parse mode from options if present
+            if (options) |options_object| {
```

I dont think this has any meaning for s3 right? we can just throw an error here or just ignore it instead of parsing it

---

+1 I would expect it to error